app/helpers/pages_helper.rb

```diff
@@ -9,7 +9,7 @@ def version
 
   def subtitle
     subtitle = "v#{version_number}"
-    subtitle += " - #{nice_date_for(version.built_at)}" if version.try(:built_at)
```

Comment:
I believe that `created_at` is made automatically on every version record by rails so it should always be true